src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+public class Disability {
+    private Long disabilityId;
+    private KeyValue type;
```

Comment:
What does 'type' mean? Is there a better description?

Reply:
Renamed to disabilityType